gateway-api/users-api/app.js

```diff
@@ -0,0 +1,20 @@
+// app.js
+// Bootstrapping the express application
+// Loads all routes
+
+const app = require('express')();
+const cors = require('cors');
+const bodyParser = require('body-parser');
+
+// Allow cross origin requests
+app.use(cors());
+
+// Allow JSON request bodies
+app.use(bodyParser.json());
+
+// Load in all of the routes:
+app.get('/alive', require('./api/alive.js')());
+app.get('/users/:id', require('./api/get-user.js')());
```

Comment:
This file should not be here. Your route should be in the users-api folder, not a new file in the gateway-api